acid/utils.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+
+
+def pipe_intersect(pipelines_config, pipelines_json):
+    if not pipelines_config or not pipelines_json:
+        return []
+
+    pipelines_to_show = [pipeline['name'] for pipeline in pipelines_json
```

Comment:
i could be just:
return [pipeline['name'] for pipeline in pipelines_json
           if pipeline['name'] in pipelines_config]

Reply:
Yeah, it's leftover from earlier version where `pipelines_to_show` was assigned to empty list in previous if statement